src/Microsoft.AspNetCore.SignalR.Client/HubConnection.cs

```diff
@@ -45,29 +46,29 @@ public class HubConnection
             remove { _connection.Closed -= value; }
         }
 
+        public HubConnection(Uri url, IInvocationAdapter adapter)
+            : this(new Connection(url), adapter, null)
+        { }
+
         public HubConnection(Uri url, IInvocationAdapter adapter, ILoggerFactory loggerFactory)
             : this(new Connection(url, loggerFactory), adapter, loggerFactory)
         { }
 
+        public HubConnection(IConnection connection, IInvocationAdapter adapter)
```

Comment:
We can remove this ctor. `IConnection` is mostly for testing (or some advanced scenarios not many people will probably try)